pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -235,6 +235,31 @@ const NET_CONFIG_TEMPLATE = `{
   }
 }`
 
+const NET_CONFIG_TEMPLATE_V6 = `{
+  "cniVersion": "0.3.0",
+  "name": "kubenet",
+  "type": "bridge",
+  "bridge": "%s",
+  "mtu": %d,
+  "addIf": "%s",
+  "isGateway": true,
+  "ipMasq": false,
+  "hairpinMode": %t,
+  "ipam": {
+    "type": "host-local",
+    "ranges": [
+    {
+     "subnet": "10.1.0.0/24",
```

Comment:
Why do we bother with IPv4 when we want IPv6?  or do you intend to do dual-stack here?  If we really want dual-stack, then we should plumb that through and not use a fake static IPv4 address.

Reply:
If the question is compatibility, since the JSON is now 0.3.0, that's not going to be compatible with some of the CNI binaries out there already.  To be fully compatible, you'd want to run a VERSION request on the bridge plugin to see what it supports, and then based on the returned array select the correct configuration JSON.

Reply:
@dcbw Thanks for the review. I will remove the IPv4.
Thanks for the suggestion on the VERSION, this seems cleaner, however, the problem I ran into is that with CNI version 0.6.0, the CNI spec version remained at 1.3.1, same as the previous. Is there some way we can detect the release version? If not is there a way to update CNI to output the release version?